src/demo-app/a11y/a11y.html

```diff
@@ -1,9 +1,12 @@
 <h1>Accessibility Demo</h1>
 
 <nav>
+  <a (click)="skipNavigation()" href="javascript:void(0)">Skip navigation</a>
```

Comment:
Fragment is not working https://github.com/angular/angular/issues/13636 
We can change this to a material button and remove `href="javascript:void(0)"`